Exercise_1.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func UpdateArray(Array [5]int, index int, value int) {
+	Array[index] = value
+}
+func UpdateWithAdress(Array *[5]int, index int, value int) {
+	Array[index] = value
+}
+
+func main1() {
```

Comment:
Why is it `main1`?

Reply:
Just because I decided to make all the exercise files in one folder, renaming the main functions in the previous files so that I wouldn't have any errors. Sorry, my bad.